### DISCUSSION TITLE
Is use of `\s5` valid?

### DISCUSSION THREAD

**Discussion by kavitharaju** [usfm-bible/tcdocs#27] [General]
Several of the UsfmjsTests in the repo contain the use of `\s5` marker? Is it a valid USFM marker? If yes, shouldn't it contain some text in it like other `\s#` markers?

- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/isa_verse_span/origin.usfm#L13
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/isa_footnote/origin.usfm#L12
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/tit_extra_space_after_chapter/origin.usfm#L17
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/1ch_verse_span/origin.usfm#L11
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfmBodyTestD/origin.usfm#L21
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/acts_1_milestone.oldformat/origin.usfm#L9
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/nb/origin.usfm#L4
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfmIntroTest/origin.usfm#L64
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfm-body-testF/origin.usfm#L27
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/out_of_sequence_verses/origin.usfm#L26
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/acts_1_milestone/origin.usfm#L11
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/luk_quotes/origin.usfm#L22
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/doo43-1/origin.usfm#L13
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/doo43-2/origin.usfm#L13

**Comment by jwickberg** [comment]
Only \s1 - \s4 are defined in the grammar. The current documentation site doesn't explicitly note that there is a limited number of markers.

I believe for syntax that empty paragraphs should be accepted. In Paratext, users would need to run a check to see that they have empty paragraphs or empty character markers.
